app/controllers/api/v1/api_tokens_controller.rb

```diff
@@ -19,15 +19,15 @@ def create
         if @api_token.save
           render json: ApiTokenSerializer.new(@api_token).as_json, status: :created
         else
-          render json: { errors: @api_token.errors }, status: :unprocessable_content
+          render json: { errors: @api_token.errors }, status: :unprocessable_entity
```

Comment:
`unprocessable_entity` is deprecated https://github.com/rspec/rspec-rails/issues/2763#issuecomment-2177371545 it is recomended to use `unprocessable_content` to comply with the standards.